src/Controller/DefaultController.php

```diff
@@ -31,33 +33,70 @@ class DefaultController extends AbstractController
      */
     public function nasIndex(FundingOrgFilter $fundingOrgFilter)
     {
-        $filter = array();
```

Comment:
Put the filter back in place the way it was, and move the foreaches below it. The filter did work because of line 36 (deleted 36)